ui/pages/confirmations/components/simulation-details/useBalanceChanges.test.ts

```diff
@@ -55,6 +59,11 @@ const dummyBalanceChange = {
   newBalance: '0xIGNORE' as Hex,
 };
 
+const pendingImpl = () =>
```

Comment:
Minor, but capitals as it's a constant, and maybe a more explicit name such as `PENDING_PROMISE`?

Reply:
Done.